tests/unit/components/pacbio/PacbioLibraryCreate.spec.js

```diff
@@ -75,21 +74,16 @@ describe('PacbioLibraryCreate.vue', () => {
       await flushPromises()
       wrapper = wrapperObj
       store = storeObj
-      modal = wrapperObj.vm
+      cmp = wrapperObj.vm
     })
 
     it('will have a Create Library button component', () => {
       const button = wrapper.findComponent('#pacbioLibraryCreate')
-      expect(button.element).toBeTruthy()
       expect(button.element.disabled).toBe(false)
     })
 
-    it('will have an modal component', () => {
```

Comment:
How was this modal check passing before 🤔 ?

Reply:
It was stubbing modal.createLibraryInTraction. Because that is a method on the store which is available on the modal it was working. When I changed it to be the same name it was then stubbing the modal method rather than the store method.

My suspicion is that is why the method name was changed.